### PR TITLE
FEATURE: SD-287 Documentation for UI SDK API

### PR DESCRIPTION
**Pull request by naibinh**
## PR Checklist
- [x] Refined and approved by Doc team

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/6/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/10/ : SUCCESS in 2m 23s

**Comment by naibinh** [comment]
@evgeniabogranskaya , please help me refine the doc.
Thanks.

Related resources:
- Images by storybooks, [link](https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1360/artifact/test-storybook/html-report/index.html), need VPN to access
- [GDP-342](https://confluence.intgdc.com/pages/viewpage.action?pageId=206196212)

**Comment by evgeniabogranskaya** [comment]
@naibinh, will do! Will post a comment here when done, thanks!

File: docs/chart_config.md [hunk 1]
```diff
@@ -297,6 +299,31 @@ import { Visualization } from '@gooddata/react-components';
 />
 ```
 
+## Configure optional stacking
```
Comment (macekond) [inline docs/chart_config.md]: I suggest renaming the section to "Stacking in charts" or just "Stacking"

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/8/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/12/ : SUCCESS in 58s

File: docs/chart_config.md (hunk 1 above)
Comment (naibinh) [reply docs/chart_config.md]: it's done

**Comment by evgeniabogranskaya** [comment]
@naibinh , I cannot push my changes to your fork, so I created a PR: https://github.com/naibinh/gooddata-ui/pull/1
Please review and merge. Thanks.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/16/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/21/ : SUCCESS in 31s

**Comment by naibinh** [comment]
Hello @evgeniabogranskaya ,

I have resolved your comments.
Please make your final review.

Thanks for that.

**Comment by evgeniabogranskaya** [comment]
@naibinh , will do!!! Thanks!

File: docs/chart_config.md [hunk 2]
```diff
@@ -297,6 +299,35 @@ import { Visualization } from '@gooddata/react-components';
...
+* To display the percentage contribution of each metric, enable `config.stackMeasuresToPercent`.
+    * If both `config.stackMeasuresToPercent` and `config.stackMeasures` are present, `config.stackMeasuresToPercent` overwrites `config.stackMeasures`.
+    * For charts with the secondary axis, `config.stackMeasuresToPercent` is applied only to the left axis. 
+
+
+```javascript
+import { Visualization } from '@gooddata/react-components';
+
+// Example of embedding a visualization with settings for optional stacking
+<Visualization
```
Comment (evgeniabogranskaya) [inline docs/chart_config.md]: Pls update to say:
Example of embedding a visualization with stacking

**Comment by evgeniabogranskaya** [comment]
@naibinh , just one small comment (sorry, missed this part yesterday) - otherwise looks good to me, thank you!!!

File: docs/chart_config.md (hunk 2 above)
Comment (naibinh) [reply docs/chart_config.md]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-opensource-repository-check/17/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/22/ : SUCCESS in 30s

**Comment by evgeniabogranskaya** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/6/ : SUCCESS in 33s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/23/ : SUCCESS in 44s

**Pull request merged**